pkg/parse/root_test.go

```diff
@@ -1788,3 +1788,145 @@ func TestSummarizeErrors(t *testing.T) {
 		})
 	}
 }
+
+func TestPrependRootSyncRemediatorStatus(t *testing.T) {
+	const rootSyncName = "my-root-sync"
+	const thisManager = "this-manager"
+	const otherManager = "other-manager"
+	conflictingObject := fake.NamespaceObject("foo-ns",
+		core.Annotation(metadata.ResourceManagerKey, otherManager))
+	conflictError := status.ManagementConflictErrorWrap(conflictingObject, thisManager)
+	invertedObject := fake.NamespaceObject("foo-ns",
+		core.Annotation(metadata.ResourceManagerKey, thisManager))
+	invertedConflictError := status.ManagementConflictErrorWrap(invertedObject, otherManager)
+	kptManagementConflictError := applier.KptManagementConflictError(conflictingObject)
+	currentManagerError := conflictError.CurrentManagerError()
+	conflictingManagerError := conflictError.ConflictingManagerError()
+	// Assert the value of each error message to make each value clear
+	const thisManagerMessage = `KNV1060: The "this-manager" reconciler cannot manage resources declared in another repository. Remove the declaration for this resource from either the current repository, or the repository managed by "other-manager".
+
+metadata.name: foo-ns
+group:
+version: v1
+kind: Namespace
+
+For more information, see https://g.co/cloud/acm-errors#knv1060`
+	const otherManagerMessage = `KNV1060: The "other-manager" reconciler cannot manage resources declared in another repository. Remove the declaration for this resource from either the current repository, or the repository managed by "this-manager".
+
+metadata.name: foo-ns
+group:
+version: v1
+kind: Namespace
+
+For more information, see https://g.co/cloud/acm-errors#knv1060`
+	const kptManagementConflictMessage = `KNV1060: The "other-manager" reconciler cannot manage resources declared in another repository. Remove the declaration for this resource from either the current repository, or the managed repository.
+
+metadata.name: foo-ns
+group:
+version: v1
+kind: Namespace
+
+For more information, see https://g.co/cloud/acm-errors#knv1060`
+	// TODO: CurrentManagerError currently seems to print an incorrect error message
```

Comment:
Yeah, I noticed this when I was working on https://github.com/GoogleContainerTools/kpt-config-sync/pull/1344

It uses the same manager in both places.